### PR TITLE
fix(listing): remove extra padding - INNO-916

### PR DESCRIPTION
**Pull request by emeryro**
# PR description

Fix horizontal alignment

## QA Checklist

In order to ensure a safe and quick review, please check that your PR follow those guidelines:

* [ ] `package.json` is up-to-date and `@ecl/[system]-base` is part of the dependencies
* [ ] I have checked the dependencies
* [ ] I have given the fractal status “ready” to my component
* [ ] I have declared `@define mycomponent` in the SCSS file
* [ ] I have specified `margin: 0;` on the CSS component
* [ ] I have provided tests
* [ ] I follow the naming guidelines
* [ ] the component supports composition
* [ ] there are no hardcoded strings (all content come from the context)
* [ ] I have filled the README.md file (at least a few lines)
* [ ] My component is listed in the root README
* [ ] My PR has the right label(s)

### CODE REVIEW

**Comment by HAL-Patch-INNO** [comment]
Deploy preview for *europa-component-library* ready!

Built with commit c6dc4a8cb9f6990e0fe618e3320a509dd616032a

https://deploy-preview-686--europa-component-library.netlify.com

File: src/generic/generic-component-listing/generic-component-listing.scss [hunk 1]
```diff
@@ -31,8 +31,12 @@
         width: 50%;
       }
 
-      .ecl-list-item__link {
+      .ecl-list-item:nth-child(2n + 0) .ecl-list-item__link {
```
Comment (kalinchernev) [inline src/generic/generic-component-listing/generic-component-listing.scss]: To be honest, it's easy to understand what's going on here without reading aside. A comment or two what are these calculations meant to represent will be helpful. There's also the `nth-of-type(odd)` and `nth-of-type(even)` syntax which makes it a bit more obvious without the calculations.

File: src/generic/generic-component-listing/generic-component-listing.scss [hunk 2]
```diff
@@ -57,10 +61,18 @@
         width: 33%;
       }
 
-      .ecl-list-item__link {
+      .ecl-list-item:nth-child(3n + 0) .ecl-list-item__link {
```
Comment (kalinchernev) [inline src/generic/generic-component-listing/generic-component-listing.scss]: Some documentation here will also be useful :)

**Review by kalinchernev — COMMENTED** [review]
My only comment would be to make the logic a bit more easy to understand, otherwise it works well :)

**Review by kalinchernev — APPROVED** [review]

**Pull request merged**